modules/compatibility/src/test/java/org/apache/ignite/compatibility/testframework/util/MavenUtils.java

```diff
@@ -263,6 +263,19 @@ private static Path resolveMavenSettingsFilePath() {
 
     /** */
     private static String resolveMavenApplicationPath() {
+
+        File currentDir = new File(System.getProperty("user.dir"));
+
+        while (currentDir != null && !new File(currentDir, "mvnw").exists()) {
+            currentDir = currentDir.getParentFile();
+        }
+
+        Path mvnwPath = Paths.get(currentDir.getAbsolutePath(), "mvnw");
+
+        if (Files.exists(mvnwPath) && Files.isExecutable(mvnwPath)) {
```

Comment:
You use 2 ways to check that file exists - in L269 `File#exists()` in this line `Files.exists()`.